.github/workflows/test_docker.yml

```diff
@@ -0,0 +1,91 @@
+# Run tests on Fedora and Ubuntu Docker images using GIFT CORP and GIFT PPA on commit
+name: test_docker
+on: [push]
+permissions: read-all
+jobs:
+  test_fedora:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: ['38']
+    container:
+      image: registry.fedoraproject.org/fedora:${{ matrix.version }}
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up container
+      run: |
+        dnf install -y dnf-plugins-core langpacks-en
+    - name: Install dependencies
+      run: |
+        dnf copr -y enable @gift/dev
+        dnf install -y @development-tools python3 python3-devel libfwps-python3 libfwsi-python3 libgzipf-python3 liblnk-python3 libolecf-python3 libregf-python3 python3-dfdatetime python3-dtfabric python3-lz4 python3-pyyaml python3-setuptools
```

Comment:
remove the libyal specific dependencies, I assume this was copied from dtformats, that use these but not this project AFAIK

Reply:
Yes, I did. I liked the way you incorporated code coverage as part of my PR. Compliments.

Reply:
> Yes, I did. I liked the way you incorporated code coverage as part of my PR. Compliments.

this is functionality codecov.io provides